internal/server/v1/observations/golden/bulk_series/all_result.json

```diff
@@ -5122,54 +5198,6 @@
           "series_by_facet": [
             {
               "series": [
-                {
```

Comment:
@Bipnabraham @manishvats2 
 Data for Count_CriminalActivities_CombinedCrime at citylevel before 2020 shold have been preserved form historical data. Can youpls check why this is deleted?